Podfile

```diff
@@ -4,14 +4,16 @@ source 'https://github.com/CocoaPods/Specs.git'
 workspace 'SwiftVoiceQuickstart'
 
 abstract_target 'TwilioVoice' do
-  pod 'TwilioVoice', '=2.0.0-beta12'
+  pod 'TwilioVoice', '2.0.0-beta13'
```

Comment:
Pre-release qualifiers are always matched exactly by CocoaPods, so I omitted the `=`.